Menubar: MenubarPassThroughOptions interface is missing submenu property

### Describe the bug

`submenu` property is missing in MenubarPassThroughOptions

`submenu` is defined in `passthrough/tailwind/index.js` for `Tailwind.menubar.submenu` , and classes are applied to the submenu as expected when running.

### Reproducer

_No response_

### PrimeReact version

10.6.3

### React version

18.x

### Language

TypeScript

### Build / Runtime

Vite

### Browser(s)

_No response_

### Steps to reproduce the behavior

The following code fails to compile in typescript

```jsx
        <Menubar
          pt={{
            submenu: {
              className: "bg-blue-500",
            },
          }}
        />
```

```log
lib/components/Banner.tsx:275:13 - error TS2769: No overload matches this call.
  Overload 1 of 2, '(props: MenubarProps): Menubar', gave the following error.
    Object literal may only specify known properties, and 'submenu' does not exist in type 'MenubarPassThroughOptions'.
  Overload 2 of 2, '(props: MenubarProps, context: any): Menubar', gave the following error.
    Object literal may only specify known properties, and 'submenu' does not exist in type 'MenubarPassThroughOptions'.

275             submenu: {
                ~~~~~~~

  node_modules/primereact/menubar/menubar.d.ts:161:5
    161     pt?: MenubarPassThroughOptions;
            ~~
    The expected type comes from property 'pt' which is declared here on type 'IntrinsicAttributes & IntrinsicClassAttributes<Menubar> & Readonly<MenubarProps>'
  node_modules/primereact/menubar/menubar.d.ts:161:5
    161     pt?: MenubarPassThroughOptions;
            ~~
    The expected type comes from property 'pt' which is declared here on type 'IntrinsicAttributes & IntrinsicClassAttributes<Menubar> & Readonly<MenubarProps>'


Found 1 error in lib/components/Banner.tsx:275

```

### Expected behavior

Typescript shouldn't complain about missing `submenu` property when it works as a passthrough option and is used by the default Tailwind theme.

--- a/components/lib/menubar/menubar.d.ts
+++ b/components/lib/menubar/menubar.d.ts
@@ -192,13 +192,11 @@ export declare class Menubar extends React.Component<MenubarProps, any> {
      * @return {HTMLDivElement} Container element
      */
     public getElement(): HTMLDivElement;
-
     /**
      * Used to get root menu element.
      * @return {HTMLElement} Root menu element
      */
     public getRootMenu(): HTMLElement;
-
     /**
      * Used to get menu button element.
      * @return {HTMLElement} Menu button element

--- a/components/lib/menubar/menubar.d.ts
+++ b/components/lib/menubar/menubar.d.ts
@@ -54,6 +54,10 @@ export interface MenubarPassThroughOptions {
      */
     label?: MenubarPassThroughType<React.HTMLAttributes<HTMLSpanElement>>;
     /**
+     * Uses to pass attributes to the submenu's DOM element.
+     */
+    submenu?: MenubarPassThroughType<React.HTMLAttributes<HTMLUListElement>>;
+    /**
      * Uses to pass attributes to the submenu icon's DOM element.
      */
     submenuIcon?: MenubarPassThroughType<React.SVGProps<SVGSVGElement> | React.HTMLAttributes<HTMLSpanElement>>;
@@ -135,16 +139,19 @@ export interface MenubarProps extends Omit<React.DetailedHTMLProps<React.HTMLAtt
      * Establishes relationships between the component and label(s) where its value should be one or more element IDs.
      */
     ariaLabelledBy?: string | undefined;
+
     /**
      * Callback to invoke when menu receives focus.
      * @param {React.SyntheticEvent} event - Browser event.
      */
     onFocus?(event: React.SyntheticEvent): void;
+
     /**
      * Callback to invoke when menu loses focus.
      * @param {React.SyntheticEvent} event - Browser event.
      */
     onBlur?(event: React.SyntheticEvent): void;
+
     /**
      * The template of trailing element.
      */
@@ -188,11 +195,13 @@ export declare class Menubar extends React.Component<MenubarProps, any> {
      * @return {HTMLDivElement} Container element
      */
     public getElement(): HTMLDivElement;
+
     /**
      * Used to get root menu element.
      * @return {HTMLElement} Root menu element
      */
     public getRootMenu(): HTMLElement;
+
     /**
      * Used to get menu button element.
      * @return {HTMLElement} Menu button element

--- a/components/lib/menubar/menubar.d.ts
+++ b/components/lib/menubar/menubar.d.ts
@@ -139,19 +139,16 @@ export interface MenubarProps extends Omit<React.DetailedHTMLProps<React.HTMLAtt
      * Establishes relationships between the component and label(s) where its value should be one or more element IDs.
      */
     ariaLabelledBy?: string | undefined;
-
     /**
      * Callback to invoke when menu receives focus.
      * @param {React.SyntheticEvent} event - Browser event.
      */
     onFocus?(event: React.SyntheticEvent): void;
-
     /**
      * Callback to invoke when menu loses focus.
      * @param {React.SyntheticEvent} event - Browser event.
      */
     onBlur?(event: React.SyntheticEvent): void;
-
     /**
      * The template of trailing element.
      */

